content/docs/command-reference/add.md

```diff
@@ -384,43 +380,37 @@ outs:
 
 ## Example: Transfer to remote storage
```

Comment:
Adding large external data (straight to remote) ?

Reply:
I agree that both "Transfer to an external cache" and "Transfer to remote storage" focus on the method rather than the problem/solution but "Adding large external data" can refer to --external or --out or --to-remote. Let me think ⌛

Reply:
Alright I re-did the titles to focus on the problem being solved, PTAL @shcheklein (at everything) when you can.